gptables/__init__.py

```diff
@@ -1,6 +1,6 @@
 from gptables.core.theme import Theme
 from gptables.core.cover import Cover
-from gptables.core.gptable import GPTable
+from gptables.core.gptable import GPTable, FormatList # FormatList for testing
```

Comment:
Does FormatList need to still be in here if it was for testing?

Reply:
Imported directly from module in test_wrappers, can be removed from __init__.py